templates/helm/values.yaml.tpl

```diff
@@ -38,7 +38,12 @@ resources:
 aws:
   # If specified, use the AWS region for AWS API calls
   region: ""
-  endpoint_url: ""
+
```

Comment:
please remove the empty newlines before and after the code comment you added in order to align with the other code comments in the file.